provider/ocm_policies_data_source.go

```diff
@@ -0,0 +1,194 @@
+	"github.com/openshift-online/ocm-sdk-go/logging"
+)
+
+const (
+	// Policy IDs from type operator roles
+	CloudCred       = "openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy"
+	CloudNetwork    = "openshift_cloud_network_config_controller_cloud_credentials_policy"
+	ClusterCSI      = "openshift_cluster_csi_drivers_ebs_cloud_credentials_policy"
+	ImageRegistry   = "openshift_image_registry_installer_cloud_credentials_policy"
+	IngressOperator = "openshift_ingress_operator_cloud_credentials_policy"
+	MachineAPI      = "openshift_machine_api_aws_cloud_credentials_policy"
+
+	// Policy IDs from type account roles
+	Installer            = "sts_installer_permission_policy"
+	Support              = "sts_support_permission_policy"
+	InstanceWorker       = "sts_instance_worker_permission_policy"
+	InstanceControlPlane = "sts_instance_controlplane_permission_policy"
+)
+
+type OcmPoliciesDataSourceType struct {
+}
+
+type OcmPoliciesDataSource struct {
+	logger       logging.Logger
+	awsInquiries *cmv1.AWSInquiriesClient
+}
+
+func (t *OcmPoliciesDataSourceType) GetSchema(ctx context.Context) (result tfsdk.Schema,
+	diags diag.Diagnostics) {
+	result = tfsdk.Schema{
+		Description: "List of rosa operator role for a specific cluster.",
+		Attributes: map[string]tfsdk.Attribute{
+			"operator_role_policies": {
+				Description: "Operator role policies.",
+				Attributes:  operatorRolePoliciesNames(),
+				Computed:    true,
+			},
+			"account_role_policies": {
+				Description: "Account role policies.",
+				Attributes:  accountRolePoliciesNames(),
+				Computed:    true,
+			},
+		},
+	}
+	return
+}
+
+func accountRolePoliciesNames() tfsdk.NestedAttributes {
+	return tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+		Installer: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		Support: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		InstanceWorker: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		InstanceControlPlane: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+	})
+}
+
+func operatorRolePoliciesNames() tfsdk.NestedAttributes {
+	return tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+		CloudCred: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		CloudNetwork: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		ClusterCSI: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		ImageRegistry: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		IngressOperator: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		MachineAPI: {
+			Type:     types.StringType,
+			Computed: true,
+		},
+	})
+}
+
+func (t *OcmPoliciesDataSourceType) NewDataSource(ctx context.Context,
+	p tfsdk.Provider) (result tfsdk.DataSource, diags diag.Diagnostics) {
+	// Cast the provider interface to the specific implementation:
+	parent := p.(*Provider)
+
+	// Get the collection of clusters:
```

Comment:
Under `/aws_inquiries/` we have multiple endpoints, but none returns a collection of clusters.
Not sure the comment is accurate.

Reply:
Fix it to `collection of aws inquiries` thanks